kwanj-k/storemanager-API
# #161204450 Add PUT and DELETE product endpoints

### What does this PR do?
Add Update and Delete endpoints for products

#### Description of Task to be Completed ?
-- Have a PUT endpoint for products
-- Have a DELETE endpoint for products
-- Add tests to test the new endpoints

### How should this be manually tested?
* After cloning the repo,git pull the branch ft-updata-delete-product-#161204450
* Install a virtual enviroment and activate it [Refer here](https://docs.python-guide.org/dev/virtualenvs/)
* Pip install the the requirements.txt
* Run the following:
  ``` export FLASK_APP='run.py' ```
  ``` export APP_SETTINGS='development' ```
   ``` pytest --cov-report term-missing --cov=app ```

 All the given tests should pass

### What are the relevant Pivotal Tracker stories?
#161204450
